vagrant/deploy.sh

```diff
@@ -42,10 +45,10 @@ fi
 
 vagrant up
 
-echo "Setting up SSH tunnel to concourse..."
-
-vagrant ssh -- -L 8080:127.0.0.1:8080 -fN
-
+if ! pgrep -f "ssh.*${VAGRANT_SSH_KEY_NAME}.*-L 8080:127.0.0.1:8080 -fN" >/dev/null; then
+  echo "Setting up SSH tunnel to concourse..."
+  vagrant ssh -- -L 8080:127.0.0.1:8080 -fN
+fi
```

Comment:
Could we at least echo a message saying it's skipping setting up a new tunnel in the other case - will make it much more debuggable if for some reason this check is spuriously exiting zero.

Reply:
Done